scripts/update_vm_versions.py

```diff
@@ -0,0 +1,78 @@
+#!/bin/python3
+
+import os
+import requests
+import re
+import yaml
+from packaging import version
```

Comment:
You should setup a requirements.txt for the external packages you require. Such as PyYAML.

Example:
```python
requests==2.18.1
PyYAML==6.0
packaging==21.3
```